### PR TITLE
Relax metadata check

### LINKED ISSUE #119: Add option to relax metadata check
[`Creator.start()`](https://github.com/openzim/python-scraperlib/blob/main/src/zimscraperlib/zim/creator.py#L133) currently enforces metadata check. That's the default behavior we want to keep.

In addition, we'd like to be able to bypass those checks (to what extend being unclear) to allow creating ZIMs with scraperlib but with non-compliant metadata.

The purpose of this being to allow non-openZIM people to create ZIM files with the metadata they want, without our publishing restrictions, using our scrapers. A first use case being the playlist mode for YT that we want to keep for others but we won't be using.

See https://github.com/openzim/youtube/issues/188

### PR DESCRIPTION
**Pull request by benoit74**
Fix #119 

## Changes
- fix CHANGELOG from previous PR #132 ...
- add a `auto_metadata_check` parameter, which default to `True` but can be disabled

One question: I had to disable validation at two places, is it intentional?

### CODE REVIEW

**Comment by benoit74** [comment]
I hate naming things, and here I have no PoV, so I love your opinion 😉

File: src/zimscraperlib/zim/creator.py [hunk 1]
```diff
@@ -106,6 +111,7 @@ def __init__(
         compression: Optional[str] = None,
         workaround_nocancel: Optional[bool] = True,  # noqa: FBT002
         ignore_duplicates: Optional[bool] = False,  # noqa: FBT002
+        disable_metadata_checks: bool = True,  # noqa: FBT001, FBT002
```
Comment (rgaudin) [inline src/zimscraperlib/zim/creator.py:114]: Should be False by default ; that's the point 😅

File: tests/zim/test_zim_creator.py [hunk 2]
```diff
@@ -507,6 +507,12 @@ def test_check_metadata(tmp_path):
         Creator(tmp_path, "").config_dev_metadata(LongDescription="T" * 5000).start()
 
 
+def test_relax_metadata(tmp_path):
+    Creator(tmp_path, "", disable_metadata_checks=False).config_dev_metadata(
```
Comment (rgaudin) [inline tests/zim/test_zim_creator.py:511]: Drop that 🍷 😉

**Comment by rgaudin** [comment]
Well, you're up for a post-PR fix 🤓

**Pull request merged**